tools/kubernetes/README.md

```diff
@@ -0,0 +1,24 @@
+### Using this tool for local development
+
+**TLDR:** Just run `./shared-cluster.sh connect`.
+
+#### Prerequisites
+1. Someone on has already run `shared-cluster.up` (ask how to change the cluster if you aren't backend).
+2. You have the `gcloud` CLI installed and authenticated.
+
+#### Connecting
+If the cluster is up, all you need to know is to run `./shared-cluster.sh connect` and your local devcluster
+will be talking to a remote GKE cluster.
+
+#### Setting the shared cluster up
+1. Run `./shared-cluster.sh up`. If you aren't on backend, please `s/backend/yourteam/g` in the script before
```

Comment:
I am not entirely sure what  this `s/backend/yourteam/g` means. Maybe clarify this a bit in the cluster setup instructions?

Reply:
I realize now that that means "Replace 'backend' with your team name <team_name> in all of the exported env variables used to create/ destroy/connect to cluster". 
I do think it'd be helpful to clarify this if it isn't common jargon, but if it is, can leave this as is!